pipeline-model-definition/pom.xml

```diff
@@ -108,6 +108,11 @@
       <version>${project.version}</version>
     </dependency>
     <dependency>
+      <groupId>org.jenkinsci.plugins</groupId>
+      <artifactId>pipeline-model-declarative-agent</artifactId>
+      <version>${project.version}</version>
```

Comment:
@abayer actually it is more your setup... as you set up the parent hierarchy.

I recommend that all poms in this repo have the *exact same version* lock step.That will probably mean that the parent pom will need to be in the hierarchy... which means that we will have to deploy it too